### DISCUSSION TITLE
How to generate abundance models?

### DISCUSSION THREAD

**Discussion by rogerio-bio** [r-a-dobson/dynamicSDM#7] [General]
Hello, thank you for this package.

I can't find how the abundance models present in the supplementary material were generated. Is there something in the brt_fit phase that I should do to generate this abundance model? How should I generate one? I'm having trouble understanding this from the tutorials. 

I would also like, if possible, how to generate the static abundance map presented in the article's supplementary material.

Thank you very much

**Comment by r-a-dobson** [comment]
Hello Rogério,

The approach fits Boosted Regression Tree models to RBQ abundance estimates (`response.col`) and associated explanatory variables (`varnames`). Fitted BRTs then project likely abundance values onto rasters of explanatory variables through space and time by applying the abundance-environment relationships identified from the BRT training data. To create static maps, this involves training BRTs using static explanatory variables and combining static rasters to project BRTs onto in this way. 

However, the approach that you should use will be specific to your abundance data and requires reading on how to best fit and parameterise Boosted Regression Tree models for that type of data (e.g. changing the `distribution` argument). Also, you are not constrained to using our brt_fit() function if you'd rather model abundance-environment relationships with another model type (e.g. Random Forests, GLMs, GAMs etc.). We appreciate that there are many alternative modelling approaches for abundance data that may be better suited to your abundance data. The downstream `dynamic_proj` function can work with any model type that predict() accepts (see https://www.rdocumentation.org/packages/stats/versions/3.6.2/topics/predict).

Best wishes,
Rachel
